src/components/victory-bar.jsx

```diff
@@ -278,9 +278,11 @@ export default class VictoryBar extends React.Component {
   }
 
   getColor(props, index) {
-    const colorScale = Array.isArray(props.colorScale) ?
+    // check for styles first
+    const colorFill = props.styles && props.styles.data ? (props.styles.data.fill) : undefined;
+    const colorScale = _.isArray(props.colorScale) ?
```

Comment:
`colorScale` should be unnecessary code to run if there's a colorFill and should be conditionally run.